models/src/main/scala/coop/rchain/models/manual/Connective.scala

```diff
@@ -0,0 +1,31 @@
+package coop.rchain.models.manual
+
+final case class Connective(
+                             connectiveInstance: Connective.ConnectiveInstance = Connective.ConnectiveInstance.Empty
```

Comment:
it's better to put `import Connective._` at the top of the file then reference object each time